src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActionsSaveButtonV2.tsx

```diff
@@ -28,7 +28,9 @@ export const ArtworkActionsSaveButtonV2: FC<ArtworkActionsSaveButtonV2Props> = (
       internalID: artwork.internalID,
       id: artwork.id,
       slug: artwork.slug,
-      title: `${artwork.title}, ${artwork.date}`,
+      title: artwork.title!,
+      year: artwork.date,
+      artists: artwork.artistNames,
```

Comment:
**suggestion**: Since there's also an `artists` array attribute on the Artwork type, thoughts on keeping the naming consistent as `artistNames` to avoid confusion?